### PR TITLE
Add voila to website

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Following the JEP on the Voilà incorporation: https://github.com/jupyter/enhancement-proposals/pull/42

### CODE REVIEW

File: css/logo-nav.css [hunk 1]
```diff
@@ -470,20 +470,17 @@ body {
...
-    -webkit-transition: all .5s ease;
-    -moz-transition: all .5s ease;
-    -ms-transition: all .5s ease;
-    -o-transition: all .5s ease;
-    transition: all .5s ease;
+    -webkit-transition: all .05s ease;
+    -moz-transition: all .05s ease;
+    -ms-transition: all .05s ease;
+    -o-transition: all .05s ease;
+    transition: all .05s ease;
```
Comment (SylvainCorlay) [inline css/logo-nav.css]: `.5s` was really too slow.

File: index.html [hunk 2]
```diff
@@ -188,7 +211,7 @@ <h3 class="section-header">Currently in use at</h3>
     </div>
 </section>
 <section id="architecture">
-    <div class="section-grey">
+    <div class="section-white">
```
Comment (SylvainCorlay) [inline index.html]: Alternating grey and white. This requires making the logos a bit darker, hence the CSS change.

File: index.html [hunk 3]
```diff
@@ -148,8 +148,31 @@ <h4>Code meets data</h4>
...
+        <div class="voila-preview">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-6">
+                        <img class="img-responsive" src="assets/voilapreview.png" id="portal" alt="examples of Voilà dashboards"/>
+                    </div>
+                    <div class="col-md-6 nb-highlight-text">
+                        <h3>Voilà turns Jupyter notebooks into standalone web applications.</h3>
+                        <h4 class="nb-desc">
+                            Voilà can convert a Jupyter Notebook into an interactive dashboard that allows you to share your work with others. It is secure and customizable, giving you control over what your readers experience.
```
Comment (tgeorgeux) [inline index.html]: "Voilà can convert a Jupyter Notebook into an interactive dashboard that allows you to share your work with others. It is secure and customizable, giving you control over what your readers experience."

Voilà helps you communicate insights, by transforming a Jupyter Notebook into a stand-alone web application you can share. It gives you control over what your readers experience in a secure and customizable interactive dashboard.

File: index.html [hunk 4]
```diff
@@ -148,8 +148,31 @@ <h4>Code meets data</h4>
...
+<section id="voila">
     <div class="section-white">
+        <div class="voila-preview">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-6">
+                        <img class="img-responsive" src="assets/voilapreview.png" id="portal" alt="examples of Voilà dashboards"/>
+                    </div>
+                    <div class="col-md-6 nb-highlight-text">
+                        <h3>Voilà turns Jupyter notebooks into standalone web applications.</h3>
```
Comment (tgeorgeux) [inline index.html]: Communicate your insights (analysis?) with Voilà.

File: index.html (hunk 3 above)
Comment (SylvainCorlay) [reply index.html]: Nice! Using your new sentence!

**Comment by SylvainCorlay** [comment]
> I don't know how we decide what goes on the front page, but I made some copy edits in the meantime. Let me know if they work for you, if not I can polish it a bit more.

Thanks for the review. This is great!

**Comment by SylvainCorlay** [comment]
cc @ellisonbg @Carreau @afshin - looking for a third party to hit the green button!

**Pull request merged**